Common/Securities/CurrencyConversion/ICurrencyPathProvider.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuantConnect.Securities.CurrencyConversion
+{
+    public interface ICurrencyPathProvider
+    {
+        CurrencyEdge AddEdge(string currencyPair, SecurityType type);
+
+        CurrencyEdge AddEdge(string baseCode, string quoteCode, SecurityType type);
+
+        CurrencyPath FindShortestPath(string startCode, string endCode);
+
+        ICurrencyPathProvider Copy();
```

Comment:
is this method required? if so, why?

Reply:
Was required - when I used that graph from Currencies.cs. So that it could be read-only.

We can remove it now, yeah.